classes/submission/PKPSubmission.inc.php

```diff
@@ -1213,11 +1213,19 @@ function isCCLicense() {
 	}
 
 	/**
-         * Get views of the submission.
-         * @return int
-         */
-        function getViews() {
-                $application = Application::getApplication();
-                return $application->getPrimaryMetricByAssoc(ASSOC_TYPE_SUBMISSION, $this->getId());
-        }
+	 * Get views of the submission.
+	 * @return int
+	 */
+	function getViews() {
+			$application = Application::getApplication();
```

Comment:
Looks like an accidental indentation change?

Reply:
(It looks like this is a correction to space-based indentation, but using too many tabs)

Reply:
Fixed